demisto_sdk/commands/common/tools.py

```diff
@@ -842,17 +885,18 @@ def get_file(
         if return_content:
             return file_content
     except IOError as e:
-        logger.error(f"Could not read file {file_path}.\nError: {e}")
+        logger.error(f"Could not read file '{file_path}': {e}")
+        logger.debug("Traceback:\n" + traceback.format_exc())
```

Comment:
I want the traceback to appear only in debug logs (which only appear in the logfile and not stdout) to not overwhelm users with a larg confusing output in case of an error.